racket/collects/racket/private/prop-template-metafunction.rkt

```diff
@@ -0,0 +1,88 @@
+(module prop-template-metafunction '#%kernel
+  (#%provide (protect template-metafunction
```

Comment:
These don't need to be protected. (I'm not sure why I protected `metafunction` and `metafunction?` originally.)